exporters/stats/prometheus/src/main/java/io/opencensus/exporter/stats/prometheus/PrometheusExportUtils.java

```diff
@@ -190,6 +190,7 @@ public Void apply(DistributionData arg) {
             List<Double> boundaries = distribution.getBucketBoundaries().getBoundaries();
             List<String> labelNamesWithLe = new ArrayList<String>(labelNames);
             labelNamesWithLe.add(LABEL_NAME_BUCKET_BOUND);
+            long sum = 0;
```

Comment:
nit: `cumulativeCount` may be a more descriptive name.

Reply:
True, I will change that.